portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -14,36 +14,36 @@
 
 package com.google.sps.servlets;
 
-import com.google.gson.Gson;
-import com.google.sps.data.Comment;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
 import java.io.IOException;
+import java.util.Date;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import java.util.ArrayList;
-import java.util.List;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+/** Servlet for adding a new comment. */
+@WebServlet("/new-comment")
+public class NewCommentServlet extends HttpServlet {
 
-  private List<Comment> commentsList = new ArrayList<>();
-
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String jsonComments = new Gson().toJson(commentsList);
-    response.setContentType("application/json;");
-    response.getWriter().println(jsonComments);
-  }
+  private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String content = request.getParameter("comment");
     String name = request.getParameter("name");
+
     if (content.length() != 0) {
```

Comment:
Is this check just so we don't add empty comments to the database? If so, let's consider making the comment section of the form in your html required: https://www.w3schools.com/tags/att_input_required.asp